storage/sql.go

```diff
@@ -0,0 +1,114 @@
+package storage
+
+import (
+	"encoding/json"
+	"log"
+	"sync/atomic"
+	"time"
+
+	"github.com/rodrigo-brito/ninjabot/model"
+	"gorm.io/gorm"
+)
+
+type SQL struct {
+	lastID int64
+	db     *gorm.DB
+}
+
+type sqlData struct {
+	Key   int64 `gorm:"primaryKey"`
+	Value string
```

Comment:
Maybe, we can use the order fields in a separeted way, and not a JSON file. Wht you think?

Reply:
Hi, I didn't separate fields because I'm not sure in the future order struct will be changed or not (if a change was made maybe we cannot use the old table until we migrate structure of the table) that why I try to store data by using key/value. but it's good if we change key/value to separated fields, fast and useful for query and readable, In GORM we can store by using json.Rawmessage but it looks like store by string too, not a real JSONB type, and if we use psql.JSONB type for store, mysql, SQLite , etc not support psql.JSONB too

By the way, I will separate fields, thanks for review kub ;)